cuda_bindings/tests/test_nvvm.py

```diff
@@ -24,56 +33,139 @@
+    "c4e11c6614013d88433884c38c4280077978077398710ce6000fed100ef4800e330c421ec2c11dce"
+    "a11c6630053d88433884831bcc033dc8433d8c033dcc788c7470077b08077948877070077a700376"
+    "788770208719cc110eec900ee1300f6e300fe3f00ef0500e3310c41dde211cd8211dc2611e663089"
+    "3bbc833bd04339b4033cbc833c84033bccf0147660077b6807376887726807378087709087706007"
+    "76280776f8057678877780875f08877118877298877998812ceef00eeee00ef5c00eec300362c8a1"
+    "1ce4a11ccca11ce4a11cdc611cca211cc4811dca6106d6904339c84339984339c84339b8c3389443"
+    "3888033b94c32fbc833cfc823bd4033bb0c30cc7698770588772708374680778608774188774a087"
+    "19ce530fee000ff2500ee4900ee3400fe1200eec500e3320281ddcc11ec2411ed2211cdc811edce0"
+    "1ce4e11dea011e66185138b0433a9c833bcc50247660077b68073760877778077898514cf4900ff0"
+    "500e331e6a1eca611ce8211ddec11d7e011ee4a11ccc211df0610654858338ccc33bb0433dd04339"
+    "fcc23ce4433b88c33bb0c38cc50a877998877718877408077a28077298815ce3100eecc00ee5500e"
+    "f33023c1d2411ee4e117d8e11dde011e6648193bb0833db4831b84c3388c4339ccc33cb8c139c8c3"
+    "3bd4033ccc48b471080776600771088771588719dbc60eec600fede006f0200fe5300fe5200ff650"
+    "0e6e100ee3300ee5300ff3e006e9e00ee4500ef83023e2ec611cc2811dd8e117ec211de6211dc421"
+    "1dd8211de8211f66209d3bbc433db80339948339cc58bc7070077778077a08077a488777708719cb"
+    "e70eef300fe1e00ee9400fe9a00fe530c3010373a8077718875f988770708774a08774d087729881"
+    "844139e0c338b0433d904339cc40c4a01dcaa11de0411edec11c662463300ee1c00eec300fe9400f"
+    "e5000000792000001e000000721e482043880c19097232482023818c9191d144a01028643c313242"
+    "8e9021a318100a00060000006b65726e656c0000230802308240042308843082400c23080431c320"
+    "04c30c045118858c04262821373bbb36973037b737ba30bab437b7b95102231d373bbbb6343917bb"
+    "32b9b9b437b7518203000000a9180000250000000b0a7228877780077a587098433db8c338b04339"
+    "d0c382e61cc6a10de8411ec2c11de6211de8211ddec11d1634e3600ee7500fe1200fe4400fe1200f"
+    "e7500ef4b08081077928877060077678877108077a28077258709cc338b4013ba4833d94c3026b1c"
+    "d8211cdce11cdc201ce4611cdc201ce8811ec2611cd0a11cc8611cc2811dd861c1010ff4200fe150"
+    "0ff4800e00000000d11000000600000007cc3ca4833b9c033b94033da0833c94433890c301000000"
+    "6120000006000000130481860301000002000000075010cd14610000000000007120000003000000"
+    "320e10228400fc020000000000000000650c00001f000000120394f0000000000300000006000000"
+    "060000004c000000010000005800000000000000580000000100000070000000000000000c000000"
+    "130000001f0000000800000006000000000000007000000000000000000000000100000000000000"
+    "00000000060000000000000006000000ffffffff00240000000000005d0c00000d00000012039467"
+    "000000006b65726e656c31352e302e376e7670747836342d6e76696469612d637564613c73747269"
+    "6e673e0000000000",
+}
+
+MINIMAL_NVVMIR_CACHE = {}
+
+
+@pytest.fixture(params=MINIMAL_NVVMIR_FIXTURE_PARAMS)
 def minimal_nvvmir(request):
-    return MINIMAL_NVVMIR_TXT if request.param == "txt" else MINIMAL_NVVMIR_BITCODE
+    for _ in range(2):
```

Comment:
Q: why do we need a loop here?

Reply:
I made this slightly more readable: commit e166860e26e0588abc1912e526fc3a08d34033e6

It's also slightly safer, in case there is a bug in the "build cache entries" code, it will not run twice.